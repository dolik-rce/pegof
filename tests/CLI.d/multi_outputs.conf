input CLI.d/a.peg
input CLI.d/b.peg
output CLI.d/multi_outputs.a.peg.tmp
output CLI.d/multi_outputs.b.peg.tmp
