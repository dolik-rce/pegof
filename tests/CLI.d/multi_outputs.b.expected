X_B <- "X" "_" "B"
